Doc/release-notes/1.1.0.txt

```diff
@@ -15,3 +15,6 @@ Package Upgrades
 
 Fixes since v1.0.0
 ------------------
+
+ * Added Visual Studio support for Caffe.
```

Comment:
I'm a little late on this, but commenting anyway.

Formatting: This is about right except we usually leave a blank line between each bullet point.

Organization: You are welcome to adapt these conventions for Fletch, but in KWIVER and MAP-Tk there are two top-level sections: Updates and Fixes.  Fixes are things that were broken in the previous release and are now fixed (most of these go on the release branch).  Updates are new features and capabilities.   I'm not sure exactly where these changes lie, but you might consider moving them up under Updates.  They don't belong under "Package Upgrades", but maybe a new sub-section like "Compiler Support".